src/treq/client.py

```diff
@@ -5,10 +5,12 @@
 
 from io import BytesIO
 
+from six import PY3
+
 from twisted.internet.interfaces import IProtocol
 from twisted.internet.defer import Deferred
 from twisted.python.components import proxyForInterface
-from twisted.python.compat import _PY3, unicode
+from twisted.python.compat import unicode
```

Comment:
six.text_type?

Reply:
You want me to rescope this up to 'stop using `twisted.python.compat`'?  `:]`

Reply:
I'd be in favour of it - eg what the output would have been if twisted used `modernize` instead of `t.p.c`

Reply:
@altendky I think it would be useful to replace all the compat shims with six shims, as then `pyupgrade` can help make this project py3+ only

Reply:
Up to you on scope really
Happy to have this in broken -> modernized -> pyupgraded
Or broken -> fixed -> modernized -> pyupgraded